### ISSUE TITLE
Docker Desktop quits after clicking on multi-container app

### ISSUE THREAD

**Issue by MichaelRushton** [docker/for-win#12567]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: A2E22C1F-0F66-4C8E-9287-B040376E26D8/20220219111936

### Actual behavior

Clicking on a multi-container app quits Docker Desktop.
(Clicking on an individual container after expanding the multi-container app works as expected.)

### Expected behavior

Clicking on a multi-container app shouldn't quit Docker Desktop.

### Information

  - Windows Version: 11 Home 21H2
  - Docker Desktop Version: 4.5.1 (74721)
  - WSL2 or Hyper-V backend?  WSL2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM: No

### Output of `& "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check`

```
[050:11:26:13.658][I] ipc.NewClient: 58bb15de-com.docker.diagnose -> \\.\pipe\dockerBackendV2 com.docker.service.exe
[goroutine 1 [running, locked to thread]:
[win/backend/pkg/service.NewClientForPath(...)
[       win/backend/pkg/service/service.go:42
[win/backend/pkg/service.NewClient({0x153f112, 0x13}, {0x0, 0x0, 0x0})
[       win/backend/pkg/service/service.go:31 +0xd0
[common/pkg/diagkit/gather/diagnose.init()
[       common/pkg/diagkit/gather/diagnose/ipc_windows.go:23 +0x11f
Starting diagnostics

[PASS] DD0027: is there available disk space on the host?
[PASS] DD0028: is there available VM disk space?
[PASS] DD0031: does the Docker API work?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0001: is the application running?
[SKIP] DD0018: does the host support virtualization?
[PASS] DD0002: does the bootloader have virtualization enabled?
[PASS] DD0017: can a VM be started?
[PASS] DD0024: is WSL installed?
[PASS] DD0021: is the WSL 2 Windows Feature enabled?
[PASS] DD0022: is the Virtual Machine Platform Windows Feature enabled?
[PASS] DD0025: are WSL distros installed?
[PASS] DD0026: is the WSL LxssManager service running?
[PASS] DD0029: is the WSL 2 Linux filesystem corrupt?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0003: is the Docker CLI working?
[PASS] DD0013: is the $PATH ok?
[PASS] DD0005: is the user in the docker-users group?
[PASS] DD0007: is the backend responding?
[PASS] DD0014: are the backend processes running?
[PASS] DD0008: is the native API responding?
[PASS] DD0009: is the vpnkit API responding?
[PASS] DD0010: is the Docker API proxy responding?
[PASS] DD0006: is the Docker Desktop Service responding?
[PASS] DD0012: is the VM networking working?
[PASS] DD0032: do Docker networks overlap with host IPs?
[SKIP] DD0030: is the image access management authorized?
[PASS] DD0033: does the host have Internet access?
No fatal errors detected.
```

### Steps to reproduce the behavior

  1. Open Docker Desktop
  2. Build multi-container app
  3. Click on multi-container app

**Comment by TimofeyBiryukov** [comment]
Same here. Also win 11 latest update, Docker Desktop 4.5.1 (74721).
I'm clicking on a compose, and docker desktop exits.

**Comment by ciegate** [comment]
Same issue for me Win 10 Pro Latest

**Comment by joeyberkovitz** [comment]
Same for me - Windows 10 - 21H1, Docker 4.5.1

**Comment by pstoragithub** [comment]
Same problem also. Docker Desktop version : 4.5.1 (74721) on Windows 10 (latest updates)

**Comment by Temepest74** [comment]
Any updates? The problem is persisting

**Comment by laurazard** [comment]
This has already been tackled internally, and will be fixed in the next Docker Desktop release. In the meantime [here is a link](https://desktop-stage.docker.com/win/main/amd64/74933/Docker%20Desktop%20Installer.exe) to an internal build where this is fixed.

**Comment by markymark501998** [comment]
@laurazard Thanks! I ended up using the VS Code Docker extension and that has been working great so far for my usage.

**Comment by mat007** [comment]
We have released Docker Desktop 4.6.0, with a fix for this issue, see https://docs.docker.com/desktop/windows/release-notes/
